domains/oopshnik.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "A personal website, for me.",
+  "repo": "https://github.com/oopshnik/oopshnik.github.io",
+  "owner": {
+    "username": "oopshnik",
+    "email": ""
```

Comment:
Can you add another social? (Like discord)

Reply:
Ok